### PR TITLE
Wazo 3330 voicemails records deletion

### PR DESCRIPTION
**Pull request by xxbillouxx**
Depends-On: https://github.com/wazo-platform/wazo-sysconfd/pull/88

### CODE REVIEW

**Comment by xxbillouxx** [comment]
recheck

File: wazo_confd/sync_db.py [hunk 1]
```diff
@@ -101,11 +102,28 @@ def main():
             session.flush()
 
 
-def remove_tenant(tenant_uuid):
+def remove_tenant(tenant_uuid, config=None):
     logger.debug('Removing tenant: %s', tenant_uuid)
-    with session_scope():
-        tenant = tenant_resources_dao.get(tenant_uuid)
-        tenant_resources_dao.delete(tenant)
+    from wazo_confd.http_server import app
```
Comment (DrPyser) [inline wazo_confd/sync_db.py:107]: is it possible to put this import at module toplevel?

File: wazo_confd/sync_db.py [hunk 2]
```diff
@@ -101,11 +102,28 @@ def main():
...
+
+    with app.app_context():
+        if config:
+            app.config = config
+        from wazo_confd import sysconfd
```
Comment (DrPyser) [inline wazo_confd/sync_db.py:112]: toplevel too?

**Review by DrPyser — CHANGES_REQUESTED** [review]

**Review by pc-m — DISMISSED** [review]

**Review by DrPyser — DISMISSED** [review]

**Comment by xxbillouxx** [comment]
recheck

**Pull request merged**